plugins/woocommerce/templates/cart/cart.php

```diff
@@ -53,7 +54,7 @@
 									sprintf(
 										'<a href="%s" class="remove" aria-label="%s" data-product_id="%s" data-product_sku="%s">&times;</a>',
 										esc_url( wc_get_cart_remove_url( $cart_item_key ) ),
-										esc_html__( 'Remove this item', 'woocommerce' ),
+										sprintf( esc_attr__( 'Remove %s', 'woocommerce' ), $product_name ),
```

Comment:
Thanks for this PR, @faisal-alvi! I didn't do a full review, but leaving a question here (maybe for @opr?):

[The Cart block uses `Remove %s from cart`](https://github.com/woocommerce/woocommerce-blocks/blob/trunk/assets/js/base/components/cart-checkout/cart-line-items-table/cart-line-item-row.tsx#L316). I don't have a strong preference between both strings, but it might be a good idea to unify them, so we reduce the work for translators. Reusing the existing one in the Cart block would also guarantee that we have available translations. What do you think?

Reply:
I think this is a great idea, `Remove %s from cart` sounds good!